### PR TITLE
Create hk

### PR DESCRIPTION
**Pull request by ericsia**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by tmasikt** [comment]
nice website

File: domains/hk
Comment (21Z) [inline domains/hk:1]: delete this file

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/hk.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ericsia",
+        "email": "medlexopm@gmail.com"
+    },
+    "record": {
+        "A": "185.27.134.137"
```
Comment (21Z) [inline domains/hk.json:7]: IP should be in an array
```suggestion
        "A": ["185.27.134.137"]
```

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/hk.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ericsia",
+        "email": "medlexopm@gmail.com"
+    },
+    "record": {
+        "A": ["185.27.134.137"]
```
Comment (Stef-00012) [inline domains/hk.json:7]: I cannot reach your website
> 185.27.134.137 didn’t send any data.

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/hk.json (hunk 2 above)
Comment (Stef-00012) [reply domains/hk.json:7]: > > I cannot reach your website
> > > 185.27.134.137 didn’t send any data.
> 
> You can't browse it directly using web browser. But the A record is correct

what will you use the domain for?

Comment (Stef-00012) [reply domains/hk.json:7]: then seems good

**Review by Stef-DP — APPROVED** [review]

File: domains/hk.json (hunk 2 above)
Comment (Stef-00012) [reply domains/hk.json:7]: no

**Pull request merged**